src/bridge-deployer/README.md

```diff
@@ -2,12 +2,22 @@
 
 Bridge Deployer is a CLI tool for deploying and managing various types of bridge contracts on the Internet Computer and Ethereum networks.
 
+## Glossary
+
+- **EVM Canister**: canister installed on Internet Computer running the **Bitfinity EVM**
+- **Principal**: A unique identifier for a canister on the Internet Computer
+- **Base token**: the native token to bridge (e.g. USDT on Ethereum).
```

Comment:
'USDT on Ethereum'? do you mean `ETH on Ethereum`?

Reply:
I've specified also another token in the example, but it's correct, the base token is like an ERC20 on the base evm to bridge on the wrapped side, for instance USDT, SHIB, WBTC, etc